### ISSUE TITLE
CallExpresssion in array cause wrong sort result

### ISSUE THREAD

**Issue by ducktordanny** [ducktordanny/eslint-plugin-ng-module-sort#1]
If we have an element in e.g. `imports` which includes a function call, the auto-fix doesn't work as expected. For instance:

```ts
TranslateModule.forRoot({
  loader: {
    provide: TranslateLoader,
    useFactory: httpTranslateLoaderFactory,
    deps: [HttpClient],
  },
}),
```

If I run auto-fix on this it's being replaced by an empty string. It should be fixed and include more test cases that check more scenarios like this.

**Comment by ducktordanny** [comment]
For implementing a fixer it may be better to use something like this:

```ts
// get source for just this AST node
var nodeSource = sourceCode.getText(node);
```
Reference: https://eslint.org/docs/latest/extend/custom-rules#getting-the-source

Should be checked later...

**Comment by ducktordanny** [comment]
Fixed with this commit: 1ed20dd9d781f4abfbd73879f8fca88638e3efb7
